rets/http/client.py

```diff
@@ -35,7 +36,7 @@ def __init__(self,
 
         splits = urlsplit(login_url)
         self._base_url = urlunsplit((splits.scheme, splits.netloc, '', '', ''))
-        self._capabilities = {
+        self._capabilities = capability_urls or {
```

Comment:
The `login_url` only provides the URL for the login transaction. We'd still need the capability url to provide URLs for other transactions (getobject, getmetadata, search, etc.)